ftw/iframeblock/tests/test_content_type.py

```diff
@@ -26,10 +26,15 @@ def test_block_can_be_added_with_factories_menu(self, browser):
         browser.visit(content_page)
 
         self.assertTrue(len(browser.css('.sl-block')), 'Expect one block')
-        self.assertEqual(
-            '<iframe width="100%" class="iframeblock loading" '
-            'onload="onIframeLoaded(this)" data-height-calculation-method="bodyOffset" '
-            'src="http://www.google.com" '
-            'scrolling="auto" height="400"></iframe>',
-            browser.css('iframe.iframeblock').first.outerHTML
-        )
+
+        from lxml import html
+        tree = html.fromstring(browser.contents)
+        iframe = tree.xpath('//iframe')[0]
+
+        self.assertEqual(iframe.get('width'), '100%')
```

Comment:
Everything looks good now, but why do you don't use ftw.testbrowser features for the assertions?
like `browser.css('iframe').first.attrib['css], 'iframeblock loading')` or `self.assertTrue(browser.css('iframe.iframeblock.loading')` ?

Reply:
ftw.testbrowser already does the lxml part for you

Reply:
Ich kannte `.attrib` nicht und kann besser antizipieren was das Resultat sein wird, wenn ich xpath brauche. Ich habe das der Leserlichkeit halber korrigiert.
Zudem habe ich `assertListEqual` eingeführt, nachdem ich die verschiedenen Parameter in eine sorted List eingefügt habe. So ist im Diff gleich ersichtlich, wo der Fehler liegt.